code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -572,6 +574,11 @@ static HookFunction initFunction([]()
 		context.SetResult<float>(*reinterpret_cast<float*>(wheelAddr + WheelHealthOffset));
 	}));
 
+	fx::ScriptEngine::RegisterNativeHandler("GET_VEHICLE_WHEEL_SURFACE_MATERIAL", makeWheelFunction([](fx::ScriptContext& context, fwEntity* vehicle, uintptr_t wheelAddr)
+	{
+		context.SetResult<unsigned char>(*reinterpret_cast<unsigned char*>(wheelAddr + (Is2060() ? WheelSurfaceMaterialOffset_2060 : WheelSurfaceMaterialOffset_1604)));
```

Comment:
uh

you know

you can just change the int-thing on init

it's not const

not constexpr

just do it

Reply:
I got tricked... I had it that way :[

Reply:
bonus points if you just use the pattern and don't have an initial value